src/RouteResolver.tsx

```diff
@@ -197,4 +197,4 @@ const RouteResolver: React.FunctionComponent<RouteComponentProps> = () => {
   )
 }
 
-export default withRouter(RouteResolver)
```

Comment:
RouteResolver was already upgraded to use hooks in the past. withRouter is unnecessary